### PR TITLE
[RFR] Add Book formatting to the doc

### PR DESCRIPTION
**Pull request by fzaninotto**
admin-on-rest-doc

### CODE REVIEW

File: docs/InputsFields.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+layout: default
+title: "admin-on-rest - Input and Field Components"
```
Comment (jpetitcolas) [inline docs/InputsFields.md]: We should mutualize the `admin-on-rest - ` title part directly in the main layout.

File: docs/_layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE HTML>
+<html lang="" >
+    <head>
+        <title>{{ page.title }}</title>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
+        <meta name="description" content="">
```
Comment (jpetitcolas) [inline docs/_layouts/default.html]: Not sure it is useful as always empty. We should either remove it, or fill a new `description` field on each documentation page for SEO.

File: docs/_layouts/default.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="stylesheet" href="css/syntax.css">
+    </head>
+    <body>
+        <div class="book with-summary font-size-2 font-family-1">
+            <div class="book-summary">
+                <nav role="navigation">
+                    <ul class="summary">
+                        <li class="header">Table of Contents</li>
+                        <li class="chapter {% if page.path == 'index.md' %}active{% endif %}" data-level="1">
```
Comment (jpetitcolas) [inline docs/_layouts/default.html]: Not sure these `data-level` attributes are used?

File: docs/_layouts/default.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE HTML>
+<html lang="" >
```
Comment (jpetitcolas) [inline docs/_layouts/default.html]: Either remove the `lang` attribute or fill it with `en-US`.

**Comment by fzaninotto** [comment]
review applied

**Pull request merged**